spec/requests/health_check_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Health Check", type: :request do
+  describe "GET /health_check" do
+    it "returns success message for current health checks" do
+      get "/health_check"
+
+      expected = "success for following checks: ['database', 'migrations', 'email']"
+      expect(response.body).to eq(expected)
+    end
+  end
+end
```

Comment:
Can we have tests for when things are broken?

Reply:
Sure - I'll add those